tests/jobs/test_local_task_job.py

```diff
@@ -159,7 +159,10 @@ def test_heartbeat_failed_fast(self, mock_getpid):
             for i in range(1, len(heartbeat_records)):
                 time1 = heartbeat_records[i - 1]
                 time2 = heartbeat_records[i]
-                self.assertGreaterEqual((time2 - time1).total_seconds(), job.heartrate)
+                # Assert that difference small enough to avoid:
+                # AssertionError: 1.996401 not greater than or equal to 2
+                delta = abs((time2 - time1).total_seconds() - job.heartrate)
+                self.assertLessEqual(delta, 0.005)
```

Comment:
What do you think about using pytest.approx?
https://docs.pytest.org/en/latest/reference.html#id13

Reply:
Here we go again... 😜 unittests have `assertAlmostEqual` that do the same thing. But I decided to be explicit, in math when you want to be sure that something is "the same thing" you assert that de difference is small enough (null set) 🤷🏼‍♂️